File: jtransc-rt-core/src/com/jtransc/simd/Float32x4Utils.java
package com.jtransc.simd;

import com.jtransc.annotation.JTranscInvisible;

@JTranscInvisible
public class Float32x4Utils {
	static public String toStringInternal(Float32x4 v) {
		return "Simd.Float32x4(" + Float32x4.getX(v) + ", " + Float32x4.getY(v) + ", " + Float32x4.getZ(v) + ", " + Float32x4.getW(v) + ")";
	}
}

File: benchmark_kotlin_mpp/src/commonMain/kotlin/simd/Float32x4Utils.kt
package simd

object Float32x4Utils {

	fun toStringInternal(v: Float32x4?): String {
		return "Simd.Float32x4(" + Float32x4.getX(v!!).toString() + ", " + Float32x4.getY(v)
			.toString() + ", " + Float32x4.getZ(
			v
		).toString() + ", " + Float32x4.getW(v).toString() + ")"
	}
}